data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1615 @@
+			`	Torza makes a gesture of respect. "Embers sharpen your sight, friends."`
+				accept
+	on complete
+		payment 250000
+		event "remnant: ssil vida outfitter"
+		conversation
+			`Gavriil is waiting at the dock when the <ship>'s hatches finish cycling. "Welcome," they gesture. "It is good to have fresh minds to share <planet> with." The Remnant onboard stream out of the ship in a rush, each carrying a heavy pack of equipment. They gesture respect, and head off in different directions. Evidently they have studied the maps prior to arrival, as they all seem to know where they are going. Several stay behind with Gavriil and help unload the rest of the equipment, some of which appears to include the core equipment of an outfitters.`
+			`	"We have made good progress since you were here last. We have managed to take over control of life support systems in several wings of rooms, and now those are run entirely from our own systems. Aside from relieving a tiny amount of load from the station's power core, it also means that the station could fail completely and we would still have safe living quarters to serve as base of operations."`
+			`	They pause, and their gestures take on a tone of disgust. "Of course, we are working to avoid such a failure. But having multiple backup systems is the key to survival in hostile conditions."`
+			`	They look around at the few Remnant who have yet to vanish into the rest of the station. "Thank you for bringing more explorers." They hand you a data crystal. "I have already transmitted this data to Viminal, but I would appreciate it if you could take this to them to ensure there are no errors or data corruption in the transmission.`
+
+
+
+mission "Remnant: Cognizance 31"
+	name "Deliver data crystal to <planet>"
+	description "Gavriil has asked you to take a data crystal to <destination>."
+	landing
+	source "Ssil Vida"
+	destination "Viminal"
+	to offer
+		has "Remnant: Cognizance 30: done"
+	on complete
+		conversation
+			`Not knowing who to deliver the data crystal to, you take it to the information desk. The Remnant currently working there gestures that they have been expecting it and thanks you for delivering it. As you leave the information desk, your commlink chimes with a message from Prefect Chilia: he would like to speak with you, and is currently in the spaceport cafeteria.`
+
+
+
+mission "Remnant: Cognizance 32"
+	name "Pickup Refined Fuel"
+	description "Chilia has asked you to pick up a refined fuel from <planet> by <date>."
+	deadline 19
+	cargo "Reserved" 58
+	illegal -1 "Your communication panel beeps insistently with a message that a scan has been detected, and thus the mission has been terminated."
+	stealth
+	infiltrating
+	source "Viminal"
+	destination "Clink"
+	to offer
+		has "Remnant: Cognizance 31: done"
+		has "license: Remnant Capital"
```

Comment:
Could use some blocked dialogue for this mission, or just automatic granting of the capital license. I _think_  you can get here without getting the capital license (granted in Salvage 3, only need up until Salvage 2 to meet requirements according to my brief look at the files).

Reply:
Hmmm... It is mainly intended as a block for people who incur the wrath of the Remnant by revealing all their stuff to the Deep, and are labeled as untrusted, and are supposed to be blocked from getting the capital ship license. Although it looks like that wasn't fully implemented in Derpy's missions. Either way, I've added a blocked text for it.

Reply:
I am planning for there to be some options for the Player to regain that trust, but it *is* intended to be a harder route. I have that planned for before I start on the next chapter.